src/Microsoft.Diagnostics.EventFlow.Core/Implementations/EventData.cs

```diff
@@ -28,6 +29,8 @@ public class EventData: IDeepCloneable<EventData>
 
         public long Keywords { get; set; }
 
+        public object EventContext { get; set; } = ContextResolver.GetRequestContext<object>();
```

Comment:
This design also has the advantage that EventFlow Core library does not need to depend on the Correlation library anymore